packages/syft/src/syft/client/client.py

```diff
@@ -589,7 +589,7 @@ def get_nested_codes(code: UserCode) -> list[UserCode]:
             if code.nested_codes is None:
                 return result
 
-            for _, (linked_code_obj, _) in code.nested_codes.items():
```

Comment:
please check this, the changed code does not do the same thing, should be value[0] instead of key?

Reply:
thx